### PR TITLE
fix: remove logs for deleted accounts, remove unused endpoints [WPB-1523]

### PR DESCRIPTION
**Pull request by tlebon**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Deletes current logs when a user removes an account to remove history of the deleted user from the computer.

### CODE REVIEW

**Review by atomrc — APPROVED** [review]

File: electron/src/lib/LocalAccountDeletion.ts [hunk 1]
```diff
@@ -77,8 +78,13 @@ export async function deleteAccount(id: number, accountId: string, partitionId?:
...
+    const relativeFilePaths = getLogFilenames();
     await fs.remove(sessionFolder);
-    logger.log(`Deleted logs folder for account "${accountId}".`);
+    relativeFilePaths.forEach(async log => {
+      const logPath = path.join(LOG_DIR, log);
+      await fs.remove(logPath);
+    });
+    logger.log(`Deleted logs folder for current account.`);
   } catch (error: any) {
     logger.error(`Failed to delete logs folder for account "${accountId}", reason: "${error.message}".`);
```
Comment (atomrc) [inline electron/src/lib/LocalAccountDeletion.ts:89]: Shouldn't we change that as well?

Comment (tlebon) [reply electron/src/lib/LocalAccountDeletion.ts:89]: In this case I guess logs probably weren't deleted so I thought good (or acceptable) to leave it here? Idk I'm open either way

Comment (tlebon) [reply electron/src/lib/LocalAccountDeletion.ts:89]: I'd be open to leave an extended warning here to tell the user to manually delete their logs, with the location. But idk if that's best

Comment (atomrc) [reply electron/src/lib/LocalAccountDeletion.ts:89]: Thing is, this log will go to the main `electron.log` isn't that correct. I guess we should never put any account ID in those logs (that's my understanding)

Comment (tlebon) [reply electron/src/lib/LocalAccountDeletion.ts:89]: So I can imagine that in some cases it makes sense to have userId logs in this file. But just to clear them when the user is deleted. But the ticket does indeed say "don't log userId"

Comment (atomrc) [reply electron/src/lib/LocalAccountDeletion.ts:89]: wait, are you removing the entire `electron.log` and `electron.old` in this PR? Might be a little overkill 🤔 
We are going to lose logs for other accounts

**Review by atomrc — APPROVED** [review]

**Pull request merged**